virt/creating_vm/virt-creating-vms-from-instance-types.adoc

```diff
@@ -20,3 +20,21 @@ include::modules/virt-about-instance-types.adoc[leveloffset=+1]
 
 include::modules/virt-common-instancetypes.adoc[leveloffset=+1]
 
+//[id="creating-manifests-virtctl_{context}"]
+//== Creating manifests by using the virtctl tool
+
+//You can use the `virtctl` CLI utility to simplify creating manifests for VMs, VM instance types, and VM preferences. For more information, see xref:../../../virt/getting_started/virt-using-the-cli-tools.adoc#vm-manifest-creation-commands_virt-using-the-cli-tools[VM manifest creation commands].
+
+//If you have a `VirtualMachine` manifest, you can create a VM from the xref:../../../virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-cli.adoc#virt-creating-vms-from-cli[command line].
```

Comment:
those are all left as comments, just wondering why are you not just removing them or adding them ?

Reply:
@RoniKishner  Removed. That part appears elsewhere.